extensions/SemanticMediaWiki/includes/specials/SMW_SpecialSMWAdmin.php

```diff
@@ -74,16 +74,17 @@ public function execute( $par ) {
 
 		$this->messageBuilder = $this->htmlFormRenderer->getMessageBuilder();
 
-		$jobQueueLookup = $mwCollaboratorFactory->newJobQueueLookup( $this->getStore()->getConnection( 'mw.db' ) );
-		$row = $jobQueueLookup->selectJobRowFor( 'SMW\RefreshJob' );
-
-		if ( $row !== false ) { // similar to Job::pop_type, but without deleting the job
-			$title = Title::makeTitleSafe( $row->job_namespace, $row->job_title );
-			$blob = (string)$row->job_params !== '' ? unserialize( $row->job_params ) : false;
-			$refreshjob = Job::factory( $row->job_cmd, $title, $blob, $row->job_id );
-		} else {
-			$refreshjob = null;
-		}
+		#$jobQueueLookup = $mwCollaboratorFactory->newJobQueueLookup( $this->getStore()->getConnection( 'mw.db' ) );
+		#$row = $jobQueueLookup->selectJobRowFor( 'SMW\RefreshJob' );
+		#
+		#if ( $row !== false ) { // similar to Job::pop_type, but without deleting the job
+		#	$title = Title::makeTitleSafe( $row->job_namespace, $row->job_title );
+		#	$blob = (string)$row->job_params !== '' ? unserialize( $row->job_params ) : false;
+		#	$refreshjob = Job::factory( $row->job_cmd, $title, $blob, $row->job_id );
+		#} else {
+		#	$refreshjob = null;
+		#}
+		$refreshjob = null; // Wikia change - we do not use job DB table for handling tasks queue (SUS-1305)
 
 		/**** Execute actions if any ****/
 		switch ( $this->getRequest()->getText( 'action' ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "action" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)